code/modules/client/preference/preferences.dm

```diff
@@ -99,6 +99,12 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	var/parallax = PARALLAX_HIGH
 	/// 2FA status
 	var/_2fa_status = _2FA_DISABLED
+	/// Do we want to force our runechat colour to be white?
+	var/force_white_runechat = FALSE
```

Comment:
Any reason for this var?

Reply:
it was added by you, 9 months ago https://github.com/ParadiseSS13/Paradise/pull/15638/files

Reply:
And removed in CDL (most likely)